### PR TITLE
Improve Docker Compose README.md

### PR DESCRIPTION
**Pull request by nemecec**
### Motivation

Docker Compose `README.md` could be better.

### Modifications

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

**Review by Jennifer88huang — APPROVED** [review]

File: docker-compose/standalone-dashboard/README.MD
Comment (Anonymitaet) [inline docker-compose/standalone-dashboard/README.MD:23]: in which folder?

File: docker-compose/standalone-dashboard/README.MD
Comment (Anonymitaet) [inline docker-compose/standalone-dashboard/README.MD:31]:
```suggestion
The following endpoints are available (assuming Docker Compose exposes ports on `localhost`):
```

File: docker-compose/standalone-dashboard/README.MD
Comment (Anonymitaet) [inline docker-compose/standalone-dashboard/README.MD:35]:
```suggestion
* `pulsar://locahost:6650` - broker binary protocol URL, for all data transferred by producers and consumers
```

**Review by Anonymitaet — COMMENTED** [review]
Many thanks for your contribution. I’ve left some comments, PTAL, thanks.

File: docker-compose/standalone-dashboard/README.MD
Comment (nemecec) [reply docker-compose/standalone-dashboard/README.MD]: Replaced "this" with "the current".

File: docker-compose/standalone-dashboard/README.MD
Comment (nemecec) [reply docker-compose/standalone-dashboard/README.MD]: Applied

File: docker-compose/standalone-dashboard/README.MD
Comment (nemecec) [reply docker-compose/standalone-dashboard/README.MD]: Applied

**Review by Anonymitaet — APPROVED** [review]

**Review by k2la — APPROVED** [review]

**Comment by Anonymitaet** [comment]
@nemecec could you please re-run the tests? After they pass successfully, we can merge this PR, thanks.

**Comment by nemecec** [comment]
> @nemecec could you please re-run the tests? After they pass successfully, we can merge this PR, thanks.

I do not have enough permissions to trigger a re-run. I pushed an empty commit to trigger it but now it is waiting for one of the maintainers to approve the re-run.

**Comment by Anonymitaet** [comment]
@nemecec I approved and the tests are running.

**Comment by nemecec** [comment]
Even if the tests fail -- surely they are not failing because of this change as this is just a documentation change.

**Pull request merged**
